lib/command.js

```diff
@@ -1101,7 +1101,9 @@ Expecting one of '${allowedValues.join("', '")}'`);
     }
 
     // Fallback to parsing the help flag to invoke the help.
-    return this._dispatchSubcommand(subcommandName, [], [this._helpLongFlag]);
+    if (this._helpLongFlag) {
+      return this._dispatchSubcommand(subcommandName, [], [this._helpLongFlag]);
```

Comment:
I think this needs to try something and not just ignore.

I suggest use `--help` when `this._helpLongFlag` undefined?

Reply:
Thinking again, should use short flag since user specifically set it up that way.

Reply:
Oh, I didn't even look at it carefully, just thought it was some deprecated stuff like in [the other place](https://github.com/tj/commander.js/pull/1930/files/f3a45fb082dc1d7a5c85c7f13d20c615d9b2b613#diff-d8db36db6bfb32591336006f1dc1adee9e2caf7b021469995aa687c5c5723d0cR2039) where only `_helpLongFlag` is checked below. My bad.